seaborn/axisgrid.py

```diff
@@ -957,6 +957,37 @@ def set_titles(self, template=None, row_template=None, col_template=None,
                 self.axes.flat[i].set_title(title, **kwargs)
         return self
 
+    def refline(self, *, x=None, y=None, color='.5', linestyle='--', **line_kws):
+        """Add a reference line(s) to each facet.
+
+        Parameters
+        ----------
+        x, y : numeric
+            Value(s) to draw the line(s) at.
+        color : :mod:`matplotlib color <matplotlib.colors>`
+            Specifies the color of the reference line(s).
```

Comment:
Should we document that passing `color=None` will use hue mapping?

Reply:
(Perhaps both here and with a comment in the ridgeplot example — it took me a minute to figure out what was going on there (but cool that it works!))